rules-reviewed/jotm/jotm.rhamt.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="jotm"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset finds JOTM JTA implementations.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[7,8)" />
+    </metadata>
+    <rules>
+        <rule id="jotm-00001">
+            <when>
+                <javaclass references="org.objectweb.jotm.{*}">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="JOTM usage" effort="2" category-id="mandatory">
+                    <message>
+                        JOTM JTA implementations are not loaded by EAP on deployment
```

Comment:
A bit more details would be good:
your message + the likely error which will occur + the solution + the reference (which you already included)